src/layouts/Root/components/Footer.tsx

```diff
@@ -10,10 +10,13 @@ import styled, { css } from 'styled-components';
 import { math } from 'polished';
 import { getColor } from '@zendeskgarden/react-theming';
 import { ReactComponent as GardenIcon } from '@zendeskgarden/svg-icons/src/26/garden.svg';
+import { Link } from './StyledNavigationLink';
 import MaxWidthLayout from 'layouts/MaxWidth';
 
-const StyledFooterItem = styled.div`
+const StyledFooterItem = styled(Link)`
   margin-right: ${p => p.theme.space.lg};
+  cursor: pointer;
```

Comment:
Is `cursor: pointer` necessary here?

Reply:
No! I wonder if I was battling with a cache issue for when these were still `div` elements. Great spot. tyty